### ISSUE TITLE
[v1.5.0] Connectivity/Loading issue

### ISSUE THREAD

**Issue by Yurlyn** [rivafarabi/deckboard#198]
For some reason my clients can connect to the server but won't actually load the boards. It just stays on the loading screen without doing anything else. The server program is allowed through my firewall, hence it can connect, but I'm not sure why it won't load. The clients are on version 1.5.2 and version client 1.4.0 can not connect to the server v1.5.0.

**Comment by rivafarabi** [comment]
Did you make sure to update both server and client app to latest version
(1.5.0 stable for server and 1.5.2 for client)?

**Comment by Yurlyn** [comment]
Yep, both the server and the clients are all on the latest versions (s1.5.0
& c1.5.2). I had this issue as well with the 1.5.0rc versions of the server.

**Comment by Yurlyn** [comment]
I think I may have found the issue. I had a button from the Power Control extension on one of my boards. No action was set on it as I uninstalled the extension since it gave me an error that looked pretty bad (Just a lot of text but I didn't actually read it). I got rid of that button and my many boards loaded in just fine. By now I am actually on version 1.6.0-rc3 of Deckboard on the pc and 1.5.3 on my Android tablet.
![image](https://user-images.githubusercontent.com/42919979/61750642-319c1600-ada6-11e9-88ff-a8fc40098a32.png)
![image](https://user-images.githubusercontent.com/42919979/61750652-352f9d00-ada6-11e9-94bc-cf7f64fbdb3b.png)
